content/it/virtualization.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Virtualizzazione
+status: completed
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+La virtualizzazione, nell'ambito del calcolo nativo su cloud, 
```

Comment:
"calcolo nativo sul cloud" mi suona molto curioso. sento sempre utilizzare la versione in inglese. manterrei quella con la traduzione in italiano tra parentesi, al massimo.